src/subgraph.js

```diff
@@ -23,6 +24,13 @@ const throwCombinedError = (filename, errors) => {
   )
 }
 
+// Define conversions from a 'blockFormat' value in the manifest to its corresponding struct name
+const blockFormatToStructName = immutable.Map({
```

Comment:
As we now have a separation of naming convention between `blockFormat` values in the manifest and their corresponding struct names, I've included a mapping between the two here. This gets used in `validateBlockFunctions()` to ensure the data structure used in the mapping matches what was defined in the manifest.